src/identity.interfaces.ts

```diff
@@ -24,6 +19,32 @@ export type IdentityPreProcessResult = {
 
 export type IdentityAPIMethod = valueof<typeof Constants.IdentityMethods>;
 
+// TODO: Consolidate this with @mparticle/web-sdk
+export type UserIdentities = {
+    customerid?: string | undefined;
+    email?: string | undefined;
+    other?: string | undefined;
+    other1?: string | undefined; // QUESTION: Why is this missing?
```

Comment:
From what I can tell looking at the backend code, `other1` doesn't seem to exist. However, in some of our old tests, we do make several references to it: [here](https://github.com/mParticle/mparticle-web-sdk/blob/efc1f7ba05a54725f09b338d3552c28aaf4acd05/test/src/tests-identity.js#L658) and [here](https://github.com/mParticle/mparticle-web-sdk/blob/efc1f7ba05a54725f09b338d3552c28aaf4acd05/test/src/tests-identities-attributes.js#L945)

I wanted to bubble up the question since part of this exercise is to highlight places where our assumptions or tests were incorrect.

Reply:
The second link is actually an `other` key with value of `other1`, so that one doesn't apply.  First test is wrong.  It's possible that in the `createModifyIdentityRequest`, it doesn't do a validation of the identity types.  So either it should, or there is some sort of validation happening prior to that in the actual code.  My gut tells me to fix the test in the first link by removing the `other1` key, but double check the above, then make the call.